File: src/tool/spring-boot-generator/output/railway/db/damageType.js
let damageType = {
	name: "damageType",
	description: "伤损类型",
	columns: {
		id: {
			name: "id",
			description: "伤损类型编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		damageNo: {
			name: "damageNo",
			description: "伤损类型编码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		damageType: {
			name: "damageType",
			description: "伤损类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		damageDesc: {
			name: "damageDesc",
			description: "伤损类型描述",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		sortNum: {
			name: "sortNum",
			description: "排序编号",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		status: {
			name: "status",
			description: "状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		isDelete: {
			name: "isDelete",
			description: "删除标记",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		operator: {
			name: "operator",
			description: "操作人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"damageNo",
	"damageType",
	"damageDesc",
	"sortNum",
	"status",
	"isDelete",
	"createTime",
	"updateTime",
	"operator"
];

function create() {
  return {};
}

damageType.columnsArray=columnsArray;
damageType.create=create;
module.exports=damageType;

File: src/tool/spring-boot-generator/output/railway/railway/src/main/java/com/railway/pojo/entity/DamageType.java
package com.railway.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("伤损类型")
@Getter
@Setter
public class DamageType  {

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("伤损类型描述")
    private String damageDesc;

    @ApiModelProperty("伤损类型编码")
    private String damageNo;

    @ApiModelProperty("伤损类型")
    private Integer damageType;

    @ApiModelProperty("伤损类型编号")
    private Integer id;

    @ApiModelProperty("删除标记")
    private Integer isDelete;

    @ApiModelProperty("操作人")
    private Integer operator;

    @ApiModelProperty("排序编号")
    private String sortNum;

    @ApiModelProperty("状态")
    private Integer status;

    @ApiModelProperty("更新时间")
    private Date updateTime;

}
